core/src/RateModel.sol

```diff
@@ -3,27 +3,59 @@ pragma solidity 0.8.17;
 
 import {FixedPointMathLib} from "solmate/utils/FixedPointMathLib.sol";
 
+import {MAX_RATE, ONE} from "src/libraries/constants/Constants.sol";
+
+interface IRateModel {
+    /**
+     * @notice Specifies the percentage yield per second for a `lender`. Need not be a pure function
+     * of `utilization`. To convert to APY: `(1 + returnValue / 1e12) ** secondsPerYear - 1`
+     * @param utilization The `lender`'s total borrows divided by total assets, scaled up by 1e18
+     * @param lender The `Lender` to examine
+     * @return The percentage yield per second, scaled up by 1e12
+     */
+    function getYieldPerSecond(uint256 utilization, address lender) external view returns (uint256);
+}
+
 /// @title RateModel
 /// @author Aloe Labs, Inc.
 /// @dev "Test everything; hold fast what is good." - 1 Thessalonians 5:21
-contract RateModel {
+contract RateModel is IRateModel {
     uint256 private constant A = 6.1010463348e20;
 
-    uint256 private constant B = 1e12 - A / 1e18;
+    uint256 private constant B = A / 1e18;
```

Comment:
Why isn't `B` still `1e12 - A / 1e18`?

Reply:
Sorry, I should've mentioned this in the PR description. Previously, `computeYieldPerSecond` returned $1+x$ with $x$ being the equivalent of APY, denominated in seconds. Now it just returned $x$ (i.e. dropping the $1 +$ part).

Before
$B + A / (1e18 - utilization) = 1e12 - (A / 1e18) + A / (1e18 - utilization) = A / (1e18 - utilization) - A / 1e18 + 1e12$

After (we _don't_ want the extract $1 +$ term, so subtract $1e12$
$A / (1e18 - utilization) - A / 1e18 + 1e12 - 1e12 = A / (1e18 - utilization) - A / 1e18$